impl/src/main/java/org/ehcache/config/builders/UserManagedCacheBuilder.java

```diff
@@ -268,7 +267,8 @@ public void close() throws Exception {
         if (resources.contains(OFFHEAP) || resources.contains(DISK)) {
           throw new RuntimeException(e);
         } else {
-          LOGGER.debug("Could not create serializers for user managed cache {}", id, e);
+          // It doesn't matter if we didn't find it for heap because we will probably not need it
+          LOGGER.debug("Could not create serializers for user managed cache '{}': {}", id, e.getMessage());
```

Comment:
I would append that info to the logged message instead of having a comment (soon to be stale) before it.